gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -964,6 +1245,106 @@ public static OperationListOption fields(OperationField... fields) {
   }
 
   /**
+   * Class for specifying address get options.
+   */
+  class AddressOption extends Option {
+
+    private static final long serialVersionUID = -5755491818692494389L;
+
+    private AddressOption(ComputeRpc.Option option, Object value) {
+      super(option, value);
+    }
+
+    /**
+     * Returns an option to specify the address' fields to be returned by the RPC call. If this
+     * option is not provided, all address' fields are returned. {@code AddressOption.fields} can be
+     * used to specify only the fields of interest. {@link Address#addressId()} is always
+     * returned, even if not specified.
+     */
+    public static AddressOption fields(AddressField... fields) {
+      return new AddressOption(ComputeRpc.Option.FIELDS, AddressField.selector(fields));
+    }
+  }
+
+  /**
+   * Class for specifying address list options.
+   */
+  class AddressListOption extends Option {
+
+    private static final long serialVersionUID = -4281322966374929346L;
+
+    private AddressListOption(ComputeRpc.Option option, Object value) {
+      super(option, value);
+    }
+
+    /**
+     * Returns an option to specify a filter to the addresses being listed.
+     */
+    public static AddressListOption filter(AddressFilter filter) {
+      return new AddressListOption(ComputeRpc.Option.FILTER, filter.toPb());
+    }
+
+    /**
+     * Returns an option to specify the maximum number of addresses returned per page.
+     */
+    public static AddressListOption pageSize(long pageSize) {
+      return new AddressListOption(ComputeRpc.Option.MAX_RESULTS, pageSize);
+    }
+
+    /**
+     * Returns an option to specify the page token from which to start listing addresses.
+     */
+    public static AddressListOption startPageToken(String pageToken) {
```

Comment:
Did we call it "startPageToken" in other modules? I think we had only "pageToken".